puppet/lib/puppet/util/puppetdb/char_encoding.rb

```diff
@@ -71,6 +71,20 @@ def self.utf8_string(str)
   def self.warn_if_changed(str, converted_str)
     if converted_str != str
       Puppet.warning "Ignoring invalid UTF-8 byte sequences in data to be sent to PuppetDB"
+
+      begin
+        fn = '/tmp/puppetdb-invalid-utf8-sequences'
```

Comment:
Also - any location for this kind of thing should be configurable, presumably in puppetdb.conf. This would also require doc changes, and if we need a new location, packaging concerns fwiw.